in_command.go

```diff
@@ -182,3 +190,19 @@ func (c *InCommand) downloadFile(url, destPath string) error {
 
 	return nil
 }
+
+func (c *InCommand) resolveTagToCommitSHA(tag string) string {
+	reference, err := c.github.GetRef(tag)
+
+	if err != nil {
+		fmt.Fprintln(c.writer, "could not resolve tag '%s' to commit: %v", tag, err)
```

Comment:
I think it'd be better to return an error in this case, rather than leave the file blank. Better to make things fail hard and early rather than propagate an invalid content somewhere that may cause problems.